pstoreds/addr_book.go

```diff
@@ -7,6 +7,7 @@ import (
 	"time"
 
 	lru "github.com/hashicorp/golang-lru"
+	base32 "github.com/whyrusleeping/base32"
```

Comment:
@Stebalien This may be a stupid question but why don't we use `multibase` for peer ids?

Reply:
These are keys in a private local db, not meant to be shared with the world. I do have a git stash using multibase, but then I decided against it because it increases complexity and db size, and reduces performance, with no apparent value in exchange.

Reply:
I think he's asking about peer IDs themselves. The answer is: multibase came later. Ideally, we'd use CIDs, really.

Reply:
Ahhh thanks @Stebalien.  @raulk Yeah I was thinking along the lines of multibase being a 'wrapper' around several base encodings so instead of having to changes out dependencies when you change encodings, you just change the encoding parameter, i.e.
```diff
- multibase.Encode(multibase.Base58BTC, data)
+ multibase.Encode(multibase.Base32, data)
```

Anyway was just a thought :)